virttest/qemu_devices/qdevices.py

```diff
@@ -1489,7 +1489,7 @@ class QIOThread(QObject):
     ["id", "poll-max-ns", "poll-grow", "poll-shrink"]
     """
 
-    def __init__(self, iothread_id, params=None, vq_bus=False):
+    def __init__(self, iothread_id, params=None):
```

Comment:
Ah, got it, here you removed vq_bus, actually in this commit, we don't need to touch qdevices.py if we do the change in 'qdevices: add QIOThreadVQbus'